src/OpenRoad.i

```diff
@@ -330,6 +330,25 @@ write_verilog_cmd(const char *filename,
   ord->writeVerilog(filename, sort, include_pwr_gnd, remove_cells);
 }
 
+void 
+partition_design_cmd(unsigned int max_num_macro, unsigned int min_num_macro,
```

Comment:
These do not belong in openroad.i
Follow the pattern for every other tool in openroad.
start by reading docs/contrib/DeveloperGuide.rst